CondCore/EcalPlugins/plugins/BuildFile.xml

```diff
@@ -0,0 +1,147 @@
+<flags CXXFLAGS="-Wno-sign-compare -Wno-unused-variable"/>
```

Comment:
hi @depasse - changing compiler flags is not allowed. Please fix the code instead.